test/Sentry.Maui.Tests/SentryMauiOptionsTests.cs

```diff
@@ -56,4 +56,30 @@ public void AttachScreenshots_Default()
         var options = new SentryMauiOptions();
         Assert.False(options.AttachScreenshot);
     }
+    [Fact]
+    public void HandlerStrategy_Default()
+    {
+        // Arrange
+        var options = new SentryMauiOptions();
+        var expected = NdkHandlerStrategy.SENTRY_HANDLER_STRATEGY_DEFAULT;
+
+        // Assert
+        // testing default state here so nothing to assert
+
+        // Act
+        Assert.Equal(expected, options.HandlerStrategy);
+    }
+    [Fact]
+    public void HandlerStrategy_set()
```

Comment:
we're just testing setters here, can we add some integration test somehow?

Can we repro this null reference to SIGSEGV issue? We have tests in the repo where we have the SDK capture something and we grab the event from the transport and asset on what was created.

We'd add tests and `#if ANDROID`. cc @jamescrosswell @vaind are DeviceTests alive? I know we had issues
If so, any points you could give @bricefriha on how to do this?

Reply:
> cc @jamescrosswell @vaind are DeviceTests alive? I know we had issues

They're working fine on the main branch. We've got issues with them when we try to make them target net8.0-android or net8.0-ios... we don't have to do that until we bump to net9.0 though.